### PR TITLE
[CIVIS-8858] bigquery oauth finally works

### PR DESCRIPTION
**Pull request by thatguyinabeanie**
https://github.com/civisanalytics/civis-dbt-docker/pull/24

### CODE REVIEW

File: dbt/README.md [hunk 1]
```diff
@@ -66,15 +67,19 @@ For BigQuery connections, the following environment variables are required depen
...
-
-#### OAuth Credentials - TODO
+- `GCP_SERVICE_ACCOUNT_KEYFILE_JSON`: The GCP Service Account JSON Credential Key File.
 
+#### OAuth Credentials
 
+- `GCP_CLIENT_ID`: The GCP OAuth Client ID.
+- `GCP_CLIENT_SECRET`: The GCP OAuth Client Secret.
+- `GCP_TOKEN_CREDENTIAL_URI`: The GCP token credential uri.
+- `DATABASE_PASSWORD`: The GCP Oauth refresh token.
```
Comment (gskifstad) [inline dbt/README.md]:
```suggestion
- `DATABASE_PASSWORD`: The GCP OAuth refresh token.
```

File: dbt/README.md [hunk 2]
```diff
@@ -66,15 +67,19 @@ For BigQuery connections, the following environment variables are required depen
 ### Shared Environment Variables
 
 - `GCP_AUTH_METHOD`: The BigQuery authentication method.
```
Comment (gskifstad) [inline dbt/README.md]:
```suggestion
- `GCP_AUTH_METHOD`: The BigQuery authentication method. The following examples expect `oauth-secrets` or  `service-account`.
```

**Review by gskifstad — COMMENTED** [review]
two tiny suggestions

**Comment by thatguyinabeanie** [comment]
https://platform-staging.civisanalytics.com/spa/#/scripts/dbt/64058123

image

**Review by gskifstad — APPROVED** [review]
LGTM

**Pull request merged**